gdtf-spec.md

```diff
@@ -1072,6 +1072,7 @@ of geometry collect are specified in [table 30](#user-content-table-30 ).
 | [MediaServerMaster](#user-content-geometry-type-media-server-master ) | Any    | Geometry that describes a master control layer of a media device.                              |
 | [Display](#user-content-geometry-type-display )                       | Any    | Geometry that describes a surface to display visual media.                                     |
 | [GeometryReference](#user-content-geometry-type-reference )           | Any    | Reference to already described geometries.                                                     |
+| [Laser](#user-content-geometry-type-laser )                           | Any    | Geometry with a laser light output.                                                            |
```

Comment:
Do we need an extra Geometry for this? Or should this be part of the Beam Geometry? There are Moving Lights with Laser source available as well...